src/components/Button/index.tsx

```diff
@@ -393,14 +395,16 @@ const FwdButton = ({
   tooltipBaseId,
   type: elementType = 'button',
   variant = 'primary',
+  as: asProp,
   ...props
 }: ButtonProps) => {
   const as = useMemo(() => {
     if (disabled) return 'button'
+    if (asProp) return asProp
     if (to || href || download) return Link
```

Comment:
Link doesn't accept `to` props anymore so we need to remove `to` from here and all our usage into Button ?

Reply:
I put something to keep the old usage of Button.

Reply:
But this will break no ? 

If we use a props `to` and we used our Link ?